tests/lib/BrowserTestCase.php

```diff
@@ -88,27 +88,6 @@ protected function setUp(): void
 	 */
 	protected function tearDown(): void
 	{
-		$status = $this->getStatus();
```

Comment:
Panther kan als het goed is zelf screenshots maken, moet nog even kijken hoe we dat moeten configureren